src/jit/gentree.h

```diff
@@ -1332,7 +1332,7 @@ struct GenTree
 
     bool OperIsMultiRegOp() const
     {
-#if !defined(LEGACY_BACKEND) && defined(_TARGET_ARM_)
+#if !defined(LEGACY_BACKEND) && defined(_TARGET_ARM_) && defined(ARM_SOFTFP)
```

Comment:
I thought `MultiRegOp` is not used on arm hardfp, But if it is used, we need to change the implementation of `gtNewPutArgReg()`. I don't know the exact reason, but it seems to makes some regression on arm hardfp now: It is generated without initialization related to `MultiRegOp` such as gtOtherReg, but it is handled as `MultiRegOp` in `lsra.cpp`.